### ISSUE TITLE
Markdown support?

### ISSUE THREAD

**Issue by DDReaper** [madskristensen/Miniblog.Core#16]
Looking through and can't see any reference to Markdown support.

Is this provided / available or a "todo"?

**Comment by madskristensen** [comment]
I did have markdown support originally in the first iterations on this project. However, there are issues with floating images as well as Windows Live Writer that makes markdown a poor choice. With that said, it is fairly easy to support markdown by just adding support in the `Post.RenderContent` method.

I don't have it on my todo because of these issues

**Comment by DDReaper** [comment]
It's good to have markdown support for using with vscode or the online editor

**Comment by DDReaper** [comment]
I believe there are pr's to add markdown to OLW as well

**Comment by jcharlesworthuk** [comment]
Is it a "no" for markdown support then?  I'd be really keen in implementing a markdown _import_ endpoint though so I can migrate old blog posts that were written in markdown.  Could use [MarkDig](https://github.com/lunet-io/markdig) or similar to convert to HTML on the server side. Is that something you'd consider?  Happy to help.

**Comment by jgauffin** [comment]
*bump* Would you accept PRs? Like being able to choose markdown (+ a side by side editor) in the settings?

**Comment by madskristensen** [comment]
@jgauffin Nah, I think that is out of scope for this blog. For instance, how are you going to use Windows Live Writer if you use markdown?

**Comment by jgauffin** [comment]
I'm not. I'm more interested in all other aspects of the engine. Atom, RSS, SEO, Image optimization, social etc.

The demo site returns 500 or the RSS/Atom feeds BTW.
